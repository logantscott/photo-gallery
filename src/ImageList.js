import React, { Component } from 'react';
import ImageItem from './ImageItem.js';

export default class ImageList extends Component {

    render() {
        return (
            <div>
                This is my Image List.
                <ImageItem />

            </div>
        )
    }
}