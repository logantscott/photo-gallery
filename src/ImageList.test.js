import React from "react";
import ImageList from "./ImageList.js";
import renderer from "react-test-renderer";

test('renders ImageList.js correctly', () => {
  const tree = renderer
    .create(<ImageList images={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});