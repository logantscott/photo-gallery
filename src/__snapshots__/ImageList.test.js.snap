// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageList.js correctly 1`] = `
<div
  className="grid-container"
  id="image-list"
/>
`;
